README.md

```diff
@@ -103,7 +106,10 @@ Last but not least encrypt the `private-extra-vars.yaml`:
 Perform deployment
 ------------------
 
-       ansible-galaxy install -r requirements.yml -f && ansible-playbook deploy-all.yml -u ubuntu --extra-vars private-extra-vars.yml
+If the device identifier your external log volume was assigned is not /dev/vdc (the default we look for), then you'll need to pass it into the command.
+
+    ansible-galaxy install -r requirements.yml -f
+    ansible-playbook deploy-all.yml -u ubuntu --extra-vars private-extra-vars.yml --extra-vars "DALITE_LOG_DOWNLOAD_VOLUME_DEVICE_ID=/dev/vdb"
```

Comment:
Though this is one way to do it, we would override the `DALITE_LOG_DOWNLOAD_VOLUME_DEVICE_ID` in private-extra-vars.yml instead.